tox.ini

```diff
@@ -49,7 +49,7 @@ commands = behave {posargs:--tags=-php}
 [testenv:cookiecutter]
 description = Isolated cookiecutter runner
 deps = cookiecutter
-changedir = /tmp
+changedir = /tmp/painless-generated-projects
```

Comment:
I think that would be unneeded in case you use the `-o /tmp/painless-generated-projects` option in the `command =` below.